app/html/index.html

```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html>
-<html lang="en" ng-app="demoApp">
+<html lang="en" ng-app="brewBuddy">
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>DEMO APP</title>
+  <title>Brew Buddy</title>
   <script src="bundle.js"></script>
   <link rel="stylesheet" href="bundle.css">
 </head>
 <body>
-  <app-main></app-main>
+  <ng-view> </ng-view>
```

Comment:
its great to see that the only tag in the the body of your app is the ng-view
this makes it so much easier to build a single page app ! Great work!